lib/Yancy/Help/Config.pod

```diff
@@ -459,6 +459,12 @@ an error. To derive which collection, these things are considered:
 
 =back
 
+Each operation infers from the HTTP method plus other information which
+method of L<Yancy::Controller::Yancy::API> it should connect to.
+The C<id_field> parameter is the C<name> of the last C<in: "path"> parameter
```

Comment:
My rationale is e.g. this situation (from https://github.com/gothinkster/realworld/blob/master/api/swagger.json#L654): `/articles/{slug}/comments/{id}`. Arguably, it's an error to even make the comment's ID be "under" the article, since a sensible relational scheme is to have each comment's ID be globally unique. However, where that uniqueness is true, the correct ID (and from the correct "collection", which is `Comment` is to use the last, that is most specific, path parameter, not the first.

What do you think?

Reply:
Ah, I was thinking of the case `/articles/{id}/{slug}` (for accessing just the article). In this case, the slug is actually worthless, but it reads well. Humph. Those are competing use-cases... But, I agree with you that the last is probably more important more often, so we'll go with this and see what happens. We may find some cases later this doesn't work for, but we'll deal with them when we get there.